WebApp-OpenIDConnect-DotNet/appsettings.json

```diff
@@ -1,21 +1,28 @@
 ﻿{
   "Authentication": {
     "AzureAdB2C": {
+      "ApiScopes": "https://fabrikamb2c.onmicrosoft.com/helloapi/demo.read",
+      "ApiUrl": "https://fabrikamb2chello.azurewebsites.net/hello",
+
       "ClientId": "90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6",
+      "ClientSecret": "v0WzLXB(uITV5*Aq",
+
       "Tenant": "fabrikamb2c.onmicrosoft.com",
+
+      "Instance": "https://login.microsoftonline.com/tfp",
```

Comment:
Any thoughts on switching over to `b2clogin.com` in this update?

Reply:
Sorry for that, I am not very familiar with `b2clogin.com` architect. The code was tested in my own domain and what shall I change here?